packages/appstore/src/types/params.types.ts

```diff
@@ -1,5 +1,5 @@
 export type TRoute = {
-    component?: React.ComponentType;
+    component?: any;
```

Comment:
```suggestion
    component?: React.ReactNode;
```

Reply:
I get errors in routes-config in appstore if i declare react types

Reply:
how about `React.ComponentType`

Reply:
Same error 😢